internal/test/types/types.go

```diff
@@ -0,0 +1,271 @@
+package types
```

Comment:
Why is this called types? This looks like the new `Windows` package and it implies that the older `Windows` package has been repurposed and needs to be renamed.